src/cmd/tools/dtest/docker/harness/resources/harness.go

```diff
@@ -18,6 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+// Package resources contain resources needed to setup docker containers for M3 tests
```

Comment:
nit: . at end of comment

Reply:
https://github.com/m3db/m3/pull/2942/commits/055ac2e5ab867f7586fed35ad505767fae9c42ec